tests/tools/tflite_comparator/src/tflite_comparator.cc

```diff
@@ -342,30 +371,19 @@ int main(const int argc, char **argv)
         break;
       case NNFW_TYPE_TENSOR_UINT8:
       case NNFW_TYPE_TENSOR_QUANT8_ASYMM:
-        matched = compareBuffersExact<uint8_t>(ref_output.uint8, output, out_idx);
+        matched = compareBuffers<uint8_t>(ref_output.uint8, output, out_idx);
```

Comment:
(optional) @hseok-oh I would like to make it clear that `compareBuffers` allows tolerance, and it uses environment variable. What about introducing `isClose` (like [`numpy.isclose`](https://numpy.org/doc/stable/reference/generated/numpy.isclose.html)), and pass the tolerance as explicit parameter.

Reply:
OK. I'll rename.